app/views/layouts/application.html.haml

```diff
@@ -9,13 +9,42 @@
   %body
     %nav.navbar.navbar-static-top.navbar-inverse
       .container
-        %nav.collapse.navbar-collapse#navbar
+        .navbar-header
+          %button.navbar-toggle.collapsed{ type: "button", data: { toggle: "collapse", target: "#navbar" }  }
```

Comment:
Hash attribute should end with one space before the closing brace<br>Line is too long. [109/80]